pdf-search/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM mozilla/sbt AS build
+COPY src /home/app/src
```

Comment:
copy `build.sbt` first, download dependencies next and then copy src and make a full build, this will improve build time for subsequent builds